profiler/src/ProfilerEngine/Datadog.Profiler.Native/GenericSampler.h

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include "AdaptiveSampler.h"
+#include "IConfiguration.h"
+
+
+// Base class for samplers that takes care of AdaptiveSampler logistics
+class GenericSampler
+{
+private:
+    static constexpr inline std::chrono::milliseconds SamplingWindow = std::chrono::milliseconds(500);
```

Comment:
since you use std::chrono, prefer include the header `#include <chrono>`

Reply:
✅ done